### DISCUSSION TITLE
Incremental scan result report

### DISCUSSION THREAD

**Discussion by dr20240304** [blacklanternsecurity/bbot#2572] [General]
I am running bbot continuously to monitor my external assets and integrated with slack webhook to get findings/vulnerabilities. But, I am getting all the findings every time after scan finishes. Segregating duplicate findings is a pain. I am not sure if incremental scan option is available where I could get only new findings on Slack. If it's not possible, can we have this as a feature.

**Comment by TheTechromancer** [comment]
Thanks for the feature request. I agree this is something we definitely need.

Right now we are hard at work on a backend for BBOT that is designed for this exact purpose -- automating ASM. The new backend will support querying multiple scans, diffing across runs so you can see which hosts/findings are new.

Stay tuned!

**Comment by domwhewell-sage** [comment]
Just linking in this issue as they are related https://github.com/blacklanternsecurity/bbot/issues/1374
